introduction.md

```diff
@@ -3,6 +3,20 @@ Hi! This is cute!
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?